dev/DynamicDependency/MsixDynamicDependency.cpp

```diff
@@ -31,6 +40,9 @@ STDAPI MddTryCreatePackageDependency(
     MddCreatePackageDependencyOptions options,
     _Outptr_result_maybenull_ PWSTR* packageDependencyId) noexcept try
 {
+    // Dynamic Dependencies doesn't support elevation
+    FAIL_FAST_HR_IF_MSG(HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED), MddCore::IsElevated() || MddCore::IsElevated(GetCurrentProcessToken()), "DynamicDependencies doesn't support elevation. See Issue #567 https://github.com/microsoft/ProjectReunion/issues/567");
```

Comment:
Consider collapsing this into a helper method like `FailFastIfElevated()`  Copy-pasta usually suggests a method.